packages/main/src/bundle.esm.ts

```diff
@@ -119,6 +119,15 @@ import Form from "./Form.js";
 import FormItem from "./FormItem.js";
 import FormGroup from "./FormGroup.js";
 import FileUploader from "./FileUploader.js";
+import Grid from "./Grid.js";
```

Comment:
The Grid already imports the following classes
```ts
import GridRow from "./GridRow.js"; // imports GridCell.js in addition
import GridHeaderRow from "./GridHeaderRow.js";
import GridHeaderCell from "./GridHeaderCell.js";
import GridSelection from "./GridSelection.js";
```

It's better to see what actually comes with the importing Grid, and what you should additionally import to get whatever in addition.

So, here we can leave only the imports that the application should add to get everything, but as the Grid already imports most of the stuff, most of the imports are obsolete.

Same for the samples, the main.js files. Then it will more clear what should be imported. However, I saw that for the existing components we also have imported f.e. ComboBoxItem, or MultiComboBoxItem, while even without these imports everything works as the ComboBox, MultiComboBox import them internally. So, for the samples would be more refactoring and we can skip it. But at least for the bundle.esm.ts we can fix it.